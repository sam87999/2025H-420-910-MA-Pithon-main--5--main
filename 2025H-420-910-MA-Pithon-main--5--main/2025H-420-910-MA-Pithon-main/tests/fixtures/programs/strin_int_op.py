# Test de Dominic Lemay
a = "allo"
x = 5

print(a + str(x))

print(a + " toi" + "!")

print("bonjour " + " toi" + "!")

print (str(x) + " " + str(5) + str(1) + "!")