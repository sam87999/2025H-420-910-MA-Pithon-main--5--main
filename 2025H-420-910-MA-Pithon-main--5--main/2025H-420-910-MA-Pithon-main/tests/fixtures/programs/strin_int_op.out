allo5
allo toi!
bonjour  toi!
5 51!
